Misc/NEWS

```diff
@@ -10,6 +10,10 @@ What's New in Python 3.7.0 alpha 1?
 Core and Builtins
 -----------------
 
+- bpo-25324: Tokens needed for parsing in python moved to C. ``COMMENT``,
+  ``NL`` AND ``ENCODING``. This way the tokens and tok_names in token.py
```

Comment:
and: lower case

in token.py: in the token module

import tokenize.py: import the tokenize module.